modules/reference/pages/rpk/rpk-cluster/rpk-cluster-partitions-transfer-leadership.adoc

```diff
@@ -0,0 +1,53 @@
+= rpk cluster partitions transfer-leadership
+
+Transfer partition leadership between brokers.
+
+This command allows you to transfer partition leadership.
+
+You can transfer only one partition leader at a time.
+
+WARNING: Redpanda tries to balance leadership distribution across brokers by default. If the distribution of leaders becomes uneven as a result of transferring leadership across brokers, the cluster may move leadership back to the original brokers automatically.
+
+== Usage
+
+[,bash]
+----
+rpk cluster partitions transfer-leadership [flags]
+----
+
+== Examples
+
+Transfer a partition leadership:
+
+```bash
+rpk cluster partitions transfer-leadership foo --partition 0:2
+```
+
+In the example above the command transfers leadership for the partition `kafka/foo/0` to broker 2. By default, it assumes the `kafka` namespace, but you can specify an internal namespace using the `{namespace}/` prefix.
+
+Equivalent command using different syntax:
+
+```bash
+rpk cluster partitions transfer-leadership --partition foo/0:2
+```
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|--format |string |Output format (json,yaml,text,wide,help) (default "text").
+
+|-h, --help |- |Help for transfer-leadership.
+
+|-p, --partition |string |Topic-partition to transfer leadership and new leader location.
+
+|--config |string |Redpanda or rpk config file; default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
+
+|-X, --config-opt |stringArray |Override rpk configuration settings; '-X help' for detail or '-X list' for terser detail.
```

Comment:
```suggestion
|-X, --config-opt |stringArray |Override rpk configuration settings; `-X help` for detail or `-X list` for terser detail.
```

Reply:
Global flags were missing an update. This is fixed now